frontend/components/App/StatusBar/Counter.tsx

```diff
@@ -0,0 +1,121 @@
+/*!
+ * This file is part of the Stooa codebase.
+ *
+ * (c) 2020 - present Runroom SL
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+import { useEffect, useState } from 'react';
+
+import { IConferenceStatus, ITimeStatus } from '@/jitsi/Status';
+import { Fishbowl } from '@/types/api-platform';
+import useTranslation from 'next-translate/useTranslation';
+
+interface Props {
+  fishbowlData: Fishbowl;
+  timeStatus: ITimeStatus;
+  conferenceStatus: IConferenceStatus;
+  isModerator: boolean;
+}
+
+export const Counter = ({ fishbowlData, timeStatus, conferenceStatus, isModerator }: Props) => {
+  const [completedTime, setCompletedTime] = useState<boolean>(false);
+  const [timeToDisplay, setTimeToDisplay] = useState<string>('Loading...');
+  const [intervalTimer, setIntervalTimer] = useState<NodeJS.Timeout>();
```

Comment:
Not sure which is the correct typing here https://www.designcise.com/web/tutorial/what-is-the-correct-typescript-return-type-for-javascripts-settimeout-function